base/ca/src/com/netscape/ca/CAService.java

```diff
@@ -874,76 +878,97 @@ private X509CertImpl issueX509Cert(
          * signed certificate timestamp (SCT) for inclusion in the SCT extension
          * in the cert to be issued.
          */
-        String method = "CAService: issueX509Cert - CT:";
+
+        // Get CT config from CS.cfg
+        SCTProcessor sct = new SCTProcessor();
         try {
-            exts = (CertificateExtensions)
-                    certi.get(X509CertInfo.EXTENSIONS);
-            logger.debug(method + " about to check CT poison");
-            Extension ctPoison = (Extension) exts.get("1.3.6.1.4.1.11129.2.4.3");
-            if ( ctPoison == null) {
-                logger.debug(method + " ctPoison not found");
-            } else {
-                logger.debug(method + " ctPoison found");
-                logger.debug(method + " About to ca.sign CT pre-cert.");
-                cert = ca.sign(certi, algname);
+            sct.init();
+        } catch (Exception e) {
+            logger.error(e.getMessage(), e);
+        }
```

Comment:
I suppose since we are still in "prototype" stage, it's okay if we hold off taking further actions on exception till later.

Reply:
Done